src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt

```diff
@@ -59,4 +59,6 @@ class VisitDto(
   @Schema(description = "The visit modified date and time", example = "2018-12-01T13:45:00", required = true)
   @field:NotBlank
   val modifiedTimestamp: LocalDateTime,
+  @Schema(description = "Date the visit was first booked or migrated", example = "2018-12-01T13:45:00", required = false)
+  val firstBookedDateTime: LocalDateTime? = null,
```

Comment:
Do we need to have this new field on `VisitDto`? (For the requirements on the ticket we only need it on `VisitPreviewDto` as that's what's used on the Visits by date page)

Reply:
this is the DTO that is returned by visit-scheduler. We then map it into a VisitPreviewDto in the orchestration service